lib/iris/tests/unit/analysis/area_weighted/test_AreaWeightedRegridder.py

```diff
@@ -247,5 +248,41 @@ def test_src_data_different_dims(self):
         )
 
 
+@tests.skip_data
+class TestLazy(tests.IrisTest):
+    # Setup
+    def setUp(self) -> None:
+        # Prepare a cube and a template
+
+        cube_file_path = tests.get_data_path(
+            ["NetCDF", "regrid", "regrid_xyt.nc"]
+        )
+        self.cube = load_cube(cube_file_path)
+
+        template_file_path = tests.get_data_path(
+            ["NetCDF", "regrid", "regrid_template_global_latlon.nc"]
+        )
+        self.template_cube = load_cube(template_file_path)
+
+        # Chunked data makes the regridder run repeatedly
+        self.cube.data = self.cube.lazy_data().rechunk((1, -1, -1))
+
+    def test_src_stays_lazy(self) -> None:
+        cube = self.cube.copy()
+        # Regrid the cube onto the template.
+        regridder = AreaWeightedRegridder(cube, self.template_cube)
+        regridder.regrid(cube)
```

Comment:
Calling a regridder is done by treating the regridder itself as a callable object i.e:
`regridder(cube)`

Reply:
What is the difference between the two modes @stephenworsley?

Reply:
I should add that `regrid` exists as a method on `Cube`, which is what is used on the other test.